### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by dasarpan007**
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `master` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `master` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `master`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](https://classroom.github.com/help/leaving-feedback-in-github)”.
</details>

### CODE REVIEW

**Comment by bryanlailey** [comment]
Hi Victoria,

Nice code, I like how it's divided into subsections using lines of % symbols. It definitely helps readability. Also quite liked the purpose and preconditions section at the beginning. Made it clear how to get started using your function. 

Some ideas:

- For the plots: Writing [Arbitrary Units (au)] could likely be simplified to [Arbitrary Units] with no loss of information and a small gain in readability. Since the upper plot already contains altitude, time, and log(signal) information, switching from a rainbow colourmap to a two-colour or monochromatic colour map could make the information easier to digest. A descriptive title, rather than just the date, would help clarify what the plots are describing. Without reading the introductory comments of the code, it isn't obvious that the plot has anything to do with Lidar. 
- If you're going to be repeating the measurements and creating plots on different days, it would be good to use the savefig function, perhaps with a filename autogenerated from the input, so that you don't have to do it manually.
- Lines 28-45 could be substantially simplified by using something like: 
n_date = datevec(date); %creates numeric vector of date elements
s_date = num2str(n_date(1:3), '%0.2d'); %converts numeric vector to string, adds leading 0s to single digit days and months
drive = 'E:\Ceil\'; %same as your code
directory = strcat(drive,s_date(1:4),' \ ',s_date(7:8),' \ ',s_date(11:12)) %concatenates string
- The dayavg and nightavg plots use the same axis and axis titles. I'm not entirely sure that it's worth doing for only two plots but you can avoid a fair bit of duplication by using the linkaxes function. If you assign the dayavg and nightavg subplots to some variable e.g. dnavg, then you can also link the axis titles by using e.g. ylabel(dnavg, 'Height (m)'). 

Code looks good. I like the use of subplots to organize the day and night average side by side so that they're easy to compare.

**Comment by BethanyRS** [comment]
I also appreciate the extra README file and descriptions throughout your code.  I haven’t actually programmed anything with MATLAB before, but I’ll try to leave some useful suggestions.

-Because I’ve never used MATLAB, I had trouble opening Sample_PLotSept12020.fig and viewing your plot.  If it’s possible, it would have been nice to have a .png file or some other file format that I can easily open as a sample output in your repository.

-On line 49, you explained what your section of code does.  However, having never used MATLAB, I still found it a bit hard to follow.  For newbies like me, it would have been nice to have a comment or new name to specify what j is.

-Though I can still follow along for the most part, you may want to expand on variable names like “d_sum” a bit more.

-I could be misunderstanding a line that already does something like this. However, if someone makes a typo inputting the date, it would be nice to have a line like “Invalid input – please follow YYYY-MM-DD format” get printed for them.

**Review by PBarmby — COMMENTED** [review]
Hi @Vpinnegar  - Can you please save the sample plot in a common format like png or jpg and include it in the repository so I can have a look? Thanks.

**Comment by Vpinnegar** [comment]
@PBarmby I put the example plot in png now. Let me know if it works!
